packages/vue-components/src/Searchbar.vue

```diff
@@ -121,7 +121,7 @@ export default {
           headingKeywords,
         } = entry;
         const keywords = entry.keywords || '';
-        const displayTitle = title || src;
+        const displayTitle = title || src.substring(0, -1 * src.reverse().indexOf('.'));
```

Comment:
Would [`.lastIndexOf()`](https://www.w3schools.com/jsref/jsref_lastindexof.asp) be helpful here?